documentation/modules/ROOT/pages/connectors/db2.adoc

```diff
@@ -0,0 +1,1503 @@
+= Debezium Connector for Db2 
+include::../_attributes.adoc[]
+:toc:
+:toc-placement: macro
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+[NOTE]
```

Comment:
Can you please add the same note on the incubating status [as we have](https://github.com/debezium/debezium/blob/master/documentation/modules/ROOT/pages/connectors/oracle.adoc) for the Oracle connector?

Reply:
Done